src/oom/oomd-util.c

```diff
@@ -164,7 +164,7 @@ int oomd_fetch_cgroup_oom_preference(OomdCGroupContext *ctx, const char *prefix)
         if (r < 0)
                 return log_debug_errno(r, "Failed to get owner/group from %s: %m", ctx->path);
 
-        if (uid == prefix_uid) {
+        if (uid == prefix_uid || uid == 0) {
```

Comment:
please avoid the `cg_get_owner()` call for the prefix_uid if we don't need it because uid == 0 holds anyway. for the majority of cgroups the uid == 0 check will hold hence it's a pretty effective optimization.

Reply:
Thanks, I'll fix this in a follow-on.